packit_service/worker/events/pagure.py

```diff
@@ -106,6 +108,29 @@ def get_base_project(self) -> GitProject:
         logger.debug(f"Base project: {project} owned by {self.base_repo_owner}")
         return project
 
+    def get_packages_config(self) -> Optional[PackageConfig]:
+        comment = self.__dict__["comment"]
+        if comment in ("/packit-stg pull-from-upstream", "/packit pull-from-upstream"):
```

Comment:
could we use here some existing parsing functionality, e.g. [this function](https://github.com/packit/packit-service/blob/e673d06b5c4888e28bf0d1afee36d6fa7153377b/packit_service/utils.py#L188)?

Reply:
I tried but I have no access to the `service` object (`events` don't have this reference and I think this is correct) and I am not able to say if I am using `/packit` or `/packit-stg` and for this reason I need to hardcode them so I thought that hardcoding all the comment was more explicit and not that worse.

I think this is a *quick & dirty hack* 😅, if we will have other exceptions like this one we would be able to choose the package location inside the specific `handler`. But to be able to do that we need to change much more code.

Reply:
I think you can just call `ServiceConfig.get_service_config().comment_command_prefix` to get it, it seems to me much better than hardcoding.